### DISCUSSION TITLE
Facility to change columns

### DISCUSSION THREAD

**Discussion by hugh-dot-m-at-rshall-dot-us** [the-djmaze/snappymail#1652] [Ideas]
Could the columns be adjustable/changeable?

For myself, I would like to have both "To" and "From" columns in the inbox, rather than just a "From" column provided when viewing the inbox and just a "To" column provided in the Sent folder view. Ideally, a multitude of columns would be available to be turned on and off individually, as was available in Roundcube:

![image](https://github.com/the-djmaze/snappymail/assets/174763548/c224d9c7-76a7-4dc9-87a7-60cba4e508c6)

Is this possible?

Thanks for the work on this, on the back of RainLoop.  It's greatly appreciated.

**Comment by the-djmaze** [comment]
This is not possible due to the way how the messages list is constructed through CSS.
It uses `display:flex` so that it will work on any screen size (responsive) and in any view layout (horizontal/vertical split view).

To see it in action open it on your PC and make the browser window narrow and wide and see everything responding.
Now try that also in roundcube.

**Reply by hugh-dot-m-at-rshall-dot-us** [reply]
Okay. Thanks anyway.
